### PR TITLE
Remove mediatype

### PR DESCRIPTION
**Pull request by sajayantony**
This tries to address #42 and fixes the examples as per artifactType guidelines #39

Signed-off-by: Sajay Antony <sajaya@microsoft.com>

### CODE REVIEW

File: examples/net-monitor-image-sbom.json [hunk 1]
```diff
@@ -1,6 +1,5 @@
 {
-  "mediaType": "application/vnd.cncf.oras.artifact.manifest.v1+json",
-  "artifactType": "example.sbom.v0",
+  "artifactType": "application/x.example.sbom.v0",
```
Comment (shizhMSFT) [inline examples/net-monitor-image-sbom.json]: `application/vnd.example.sbom.v0`?

File: scenarios.md [hunk 2]
```diff
@@ -124,8 +123,7 @@ The SBoM content would be persisted as one or more `[blobs]` with a `subject` re
 - **tag**: _-none-_
   ```json
   {
-    "mediaType": "application/vnd.cncf.oras.artifact.manifest.v1+json",
-    "artifactType": "example.sbom.v0",
+    "artifactType": "application/x.example.sbom.v0",
```
Comment (shizhMSFT) [inline scenarios.md]: What does the `x` in `application/x.example.sbom.v0` mean?

File: specs-go/v1/manifest.go [hunk 3]
```diff
@@ -17,9 +17,6 @@ package v1
 // Manifest describes an ORAS artifact.
 // This structure provides `application/vnd.oras.artifact.manifest.v1+json` mediatype when marshalled to JSON.
```
Comment (aviral26) [inline specs-go/v1/manifest.go:18]: This should be removed

**Review by aviral26 — COMMENTED** [review]
Looks good overall

File: specs-go/v1/manifest.go (hunk 3 above)
Comment (sajayantony) [reply specs-go/v1/manifest.go:18]: I'm not following why we would remove this. My understanding was that the overall structure has the content type defined by ``application/vnd.oras.artifact.manifest.v1+json`

Comment (aviral26) [reply specs-go/v1/manifest.go:18]: It's a bit confusing to use the term media type when the spec does not define it

Comment (sajayantony) [reply specs-go/v1/manifest.go:18]: This seems to be convention that already exists - https://github.com/opencontainers/image-spec/blob/main/specs-go/v1/manifest.go#L19

**Review by shizhMSFT — DISMISSED** [review]
LGTM

File: artifact-manifest.md [hunk 4]
```diff
@@ -33,15 +33,11 @@ The high-level differences with the `oras.artifact.manifest` and the `oci.image.
```
Comment (michaelb990) [inline artifact-manifest.md:12]: Let's add a sentence in the first section (above) that says something like:

> This defines the `application/vnd.cncf.oras.artifact.manifest.v1+json` media type. 

To replace the statements below about what the `mediaType` should be for this document. This is similar to the [image-spec](https://github.com/opencontainers/image-spec/blob/main/manifest.md) wording.

**Comment by SteveLasker** [comment]
To cleanup formatting, we can remove the differences between artifacts and image-spec as they're in the root `./readme.md`

File: artifact-manifest.md [hunk 5]
```diff
@@ -33,15 +33,11 @@ The high-level differences with the `oras.artifact.manifest` and the `oci.image.
...
 
-- **`mediaType`** *string*
-
-  This field contains the `mediaType` of this document, differentiating from [image-manifest][oci-image-manifest-spec] and [image-index][oci-image-index]. The `mediaType` for this manifest type MUST be `application/vnd.cncf.oras.artifact.manifest.v1+json`, where the version WILL change to reflect newer versions.
-   
 - **`artifactType`** *string*
 
   The REQUIRED `artifactType` is a unique value, as registered with [iana.org][registering-iana].
   The `artifactType` values are equivalent to the values used in the `manifest.config.mediaType` in [OCI Artifacts][oci-artifacts].
-  Examples include `application/x.example.sbom.v0`, `application/vnd.cncf.notary.v2`.
```
Comment (SteveLasker) [inline artifact-manifest.md:44]: replacing with `sbom/example` to follow https://www.rfc-editor.org/rfc/rfc4735.html

File: artifact-manifest.md (hunk 4 above)
Comment (sajayantony) [reply artifact-manifest.md:12]: Yes. Will update that. Also realized that the differences are duplicated and worked with @SteveLasker to update that as well. 
I'm also going to update the example to reflect 
https://www.rfc-editor.org/rfc/rfc4735.html

**Comment by sajayantony** [comment]
@shizhMSFT - I had to change example mediaTypes to reflect - https://www.rfc-editor.org/rfc/rfc4735.html.

**Review by SteveLasker — APPROVED** [review]
LGTM

**Review by michaelb990 — APPROVED** [review]

**Comment by vbatts** [comment]
😸 I know it's with mixed feelings about having things be self describing. I appreciate the consistency too

**Comment by vbatts** [comment]
😸 I know it's with mixed feelings about having things be self describing. I appreciate the consistency too

**Pull request merged**